monty-31/src/x86_64_avx512/packing.rs

```diff
@@ -193,18 +293,23 @@ fn movehdup_epi32(a: __m512i) -> __m512i {
 fn mask_movehdup_epi32(src: __m512i, k: __mmask16, a: __m512i) -> __m512i {
     // The instruction is only available in the floating-point flavor; this distinction is only for
     // historical reasons and no longer matters. We cast to floats, do the thing, and cast back.
+
+    // Annoyingly, when inlined into the mul function, this seems to compile 
+    // to a vpermt2ps which has worse latency, see https://godbolt.org/z/489aaPhz3.
+    // 
+    // Hopefully this should be only a negligible different to throughput and so we don't
```

Comment:
Nit:
```suggestion
    // Hopefully this should be only a negligible difference to throughput and so we don't
```
This comment and it's mistake are copy-pasted several times below.